contracts/AutomatedAPIConsumer.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "./dev/ocr2dr/OCR2DRClient.sol";
+// import "@chainlink/contracts/src/v0.8/dev/ocr2dr/OCR2DRClient.sol"; // Once published
+import "@chainlink/contracts/src/v0.8/interfaces/KeeperCompatibleInterface.sol";
+import "@chainlink/contracts/src/v0.8/ConfirmedOwner.sol";
+import "hardhat/console.sol";
+
+/**
+ * @title Automated API Consumer contract
+ * @notice This contract is a demonstration of using OCR2DR along with Chainlink Automation.
+ * @notice NOT FOR PRODUCTION USE
+ */
+contract AutomatedAPIConsumer is OCR2DRClient, KeeperCompatibleInterface, ConfirmedOwner {
```

Comment:
`OCR2DRClient` is confusing. Maybe something like `DONDirectRequester`. It abstracts the method happening on the back end `OCR`, while saying what this contract `is`. 

It makes direct requests to a DON (decentralized oracle network).

Reply:
We should remove this contract altogether. It's just combing two Chainlink services and is confusing IMO.